tetris.js

```diff
@@ -197,85 +219,92 @@ module.exports = {
 		]
 		
 	*/
-	buildField: function(stringField){
-		
+	buildField: function (stringField) {
+
 		let arr = stringField.split("\n")
-		for(let k = 0; k < arr.length; k++){
-			for(let z = 0; z < arr[k].length; z++){
-				if( arr[k][z] != "-" && arr[k][z] != "X" && arr[k][z] != false) {
+		for (let k = 0; k < arr.length; k++) {
+			for (let z = 0; z < arr[k].length; z++) {
+				if (arr[k][z] != "-" && arr[k][z] != "X" && arr[k][z] != false) {
 					throw new Error("invalid symbol")
 				}
 			}
 		}
-		
+
 		let newArr = []
-		
-		for(let i = 0; i < arr.length; i++){
+
+		for (let i = 0; i < arr.length; i++) {
 			let elementsForNewArray = []
-			for(let j = 0; j < arr[i].length; j++){
-			if(arr[i][j] == "-"){
-				elementsForNewArray.push(false)
-			}else if(arr[i][j] == "X"){
-				elementsForNewArray.push(true)
-			 }
+			for (let j = 0; j < arr[i].length; j++) {
+				if (arr[i][j] === "-") {
+					elementsForNewArray.push(false)
+				} else if (arr[i][j] === "X") {
+					elementsForNewArray.push(true)
+				}
 			}
-			if(elementsForNewArray.length != 0){
+			if (elementsForNewArray.length != 0) {
 				newArr.push(elementsForNewArray)
 			}
 		}
-		if(newArr.length == 0){
-			throw new Error("field whitespace")		
+		if (newArr.length === 0) {
+			throw new Error("field whitespace")
 		}
 		return newArr;
 	},
-	
-	buildTetromino: function(stringOfCoordinates){
-		let math = stringOfCoordinates.split("\n")
-		let arrayOfCoordinates = []
+
+	getCoordinates: function (stringOfCoordinates) {
+		let fieldArray = stringOfCoordinates.split("\n")
 		let quantityElem = 0;
-		
-		for(let i = 0, elem_Y = 0; i < math.length; i++){
-			if(math[i]!= false){
-				
-				for(let j = 0, elem_X = 0; j < math[i].length; j++){
+		let arrayOfCoordinates = []
+
+		for (let i = 0, axisIndex_Y = 0; i < fieldArray.length; i++) {
+			if (fieldArray[i] !== "") {
+
+				for (let j = 0, axisIndex_X = 0; j < fieldArray[i].length; j++) {
 					let obj = {}
-					if(math[i][j]!= false){
-						if(math[i][j] == "X"){
-							console.log(math[i][j])
-							obj.x = elem_X
-							obj.y = elem_Y
+					if (fieldArray[i][j]=== "-" || fieldArray[i][j]=== "X") {
+						if (fieldArray[i][j] === "X") {
```

Comment:
you have 2 nested checks 
`fieldArray[i][j] === "X"`

Please rewrite so you are only checking this once